README.md

```diff
@@ -12,6 +12,10 @@ To build the Docker image for the tool, run:
 docker build . -t robopages  
 ```
 
+Optionally, you can create a bash alias like so:
+
+`alias robopages='docker run -v ~/.robopages:/root/.robopages -p 8080:8080 robopages'`
```

Comment:
hey @GangGreenTemperTatum this is a great idea! can you also add the docker socket as described here https://github.com/dreadnode/robopages-cli?tab=readme-ov-file#note-about-docker and change the port to 8000? thanks a ton buddy!

Reply:
thank you and absolutely, good call! :) apologies for the pain, i created a new branch under #7 due to some weird branching problems locally but hopefully should be good to go!